pkg/handlers/consumer.go

```diff
@@ -61,8 +63,12 @@ func (h consumerHandler) Patch(w http.ResponseWriter, r *http.Request) {
 			if err != nil {
 				return nil, err
 			}
-
-			//patch a field
+			if patch.Labels != nil {
+				found.Labels = db.EmptyMapToNilStringMap(patch.Labels)
+			}
+			if patch.Name != nil {
+				found.Name = util.EmptyStringToNil(*patch.Name)
+			}
```

Comment:
What is use case to change the consumer name here?

Reply:
it is mentioned as a valid patch field in the `ConsumerPatchRequest` - https://github.com/openshift-online/maestro/blob/e515b02da7105a46720a3924c21215b4a2894fb1/openapi/openapi.yaml#L554

i'm not sure the name should be changable, but if not, the field should be removed from the `ConsumerPatchRequest`openapi type.

Reply:
I do not know where to use consumer name. @markturansky Do you have use case to use consumer name?

Reply:
@geoberle Can you help to remove the name field from `ConsumerPatchRequest` openapi type? Thanks.

Reply:
i completely missed this comment. i will work on it now

Reply:
@clyang82 @morvencao can you allow the PR check to run?

Reply:
Consumer name isn't required, as Maestro uses the ID and expects clients to post the consumer ID